Procfile.dev

```diff
@@ -1,2 +1,2 @@
-web: unset PORT && bin/rails-sandbox server
-installer: bin/rails-sandbox g solidus_stripe:install --force --watch
+web: unset PORT && env RUBY_DEBUG_OPEN=true bin/rails-sandbox server
```

Comment:
What about a comment with usage instructions here?

Reply:
Great call, added instructions to the readme under _Development_ so it's even more visible.